package.json

```diff
@@ -61,8 +56,8 @@
     "@types/jest": "^27.0.1",
     "@types/node": "16.6.1",
     "@types/sharp": "^0.30.2",
-    "@typescript-eslint/eslint-plugin": "^4.19.0",
-    "@typescript-eslint/parser": "^4.19.0",
+    "@typescript-eslint/eslint-plugin": "^5.36.2",
+    "@typescript-eslint/parser": "^5.36.2",
```

Comment:
Updated these because eslint was complaining about this package being too old for our typescript version